.github/workflows/ci.yml

```diff
@@ -53,7 +54,7 @@ jobs:
             source $CONDA/etc/profile.d/conda.sh
             conda activate ci;
           fi
-          pytest --cov --cov-report=xml
+          pytest --cov --cov-report=xml -n 4 --dist loadscope
```

Comment:
is `n=4` chosen for a particular reason? Would `n=auto` be better?   If it's for a specific reason, maybe add a comment as to why.

Reply:
github actions provides two physical cores, aince our tests also do some IO i used more threads than available cores, but I havent measured it.

fine with using auto